client/src/components/layouts/learn.tsx

```diff
@@ -88,6 +88,8 @@ class LearnLayout extends Component<LearnLayoutProps> {
           <meta content='noindex' name='robots' />
         </Helmet>
         <main id='learn-app-wrapper'>{children}</main>
+        {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+        /* @ts-ignore  */}
```

Comment:
Migration of `DonationModal.tsx` is required to fix hence added this a last option as per guidelines